angr/calling_conventions.py

```diff
@@ -56,6 +56,11 @@ def size(self):
 
 
 class SimFunctionArgument:
+    """
+    Represent a generic function argument.
+
+    :ivar int size:    The size of the argument, in number of bytes.
```

Comment:
Is this right to speak in "number of bytes"?
Wouldn't "machine-word" be more correct?

Poke @ltfish @rhelmot .

Reply:
Machine words are some number of bytes. I can't think of anything in angr which counts in number of machine words. This particular field is definitely number of bytes.